packages/dd-trace/src/format.js

```diff
@@ -67,6 +67,9 @@ function extractTags (trace, span) {
   const priority = context._sampling.priority
 
   if (tags['span.kind'] && tags['span.kind'] !== 'internal') {
+    if (tags['span.kind'] in ['client', 'producer']) {
```

Comment:
This seems to be the opposite of what it should do.

Reply:
Hi, this file was updated in the most recent commit as you are correct, yes, the logic here is wrong. The correct logic is that the language tag should not be set when span.kind is either 'client' or 'producer'

Reply:
Was there an update to the RFC? When this was implemented the requirement was that it should be on every span that has the same service name as the process, otherwise things like correlation with runtime metrics would stop working.